lib/crypt.cpp

```diff
@@ -718,8 +719,10 @@ char *check_validity(
     int rbytes;
     unsigned char md5_md[MD5_DIGEST_LENGTH],  rbuf[2048];
 
+#if OPENSSL_VERSION_NUMBER < 0x10100000L || defined(HAVE_LIBRESSL)
```

Comment:
Please put comment with string representation of version for better reading

Reply:
Done